.github/workflows/aro-hcp-dev-env-cd.yml

```diff
@@ -130,6 +130,41 @@
                 --parameters grantCosmosAccess=false \
                 --parameters githubActionsPrincipalID=${{ secrets.GHA_PRINCIPAL_ID }}
 
+              # Azure Monitor Workspace / Grafana
+              az deployment group create \
+                --name "aro-hcp-metrics" \
+                --resource-group ${REGIONAL_RESOURCEGROUP} \
+                --template-file templates/Metrics.bicep
+
+              # enable metrics for svc cluster
+              az deployment group create \
+                --resource-group ${SC_RESOURCEGROUP} \
+                --name aks-metrics-svc-cluster-${GITHUB_RUN_ID} \
+                --template-file templates/aks-azure-monitor-metrics.bicep \
+                --parameters \
+                  azureMonitorWorkspaceResourceId=$(az monitor account show --resource-group ${REGIONAL_RESOURCEGROUP} --name aro-hcp-monitor --query id -o tsv) \
+                  azureMonitorWorkspaceLocation=$(az monitor account show --resource-group ${REGIONAL_RESOURCEGROUP} --name aro-hcp-monitor --query location -o tsv) \
+                  clusterResourceId=$(az deployment group show --resource-group "${SC_RESOURCEGROUP}" --name ${SVC_CLUSTER_NAME} --query properties.outputs.aksClusterResourceId.value -o tsv) \
+                  clusterLocation=${REGION}
+
+              # enable metrics for mgmt cluster
+              az deployment group create \
+                --resource-group ${MC_RESOURCEGROUP} \
+                --name aks-metrics-mgmt-cluster-${GITHUB_RUN_ID} \
+                --template-file templates/aks-azure-monitor-metrics.bicep \
+                --parameters \
+                  azureMonitorWorkspaceResourceId=$(az monitor account show --resource-group ${REGIONAL_RESOURCEGROUP} --name aro-hcp-monitor --query id -o tsv) \
+                  azureMonitorWorkspaceLocation=$(az monitor account show --resource-group ${REGIONAL_RESOURCEGROUP} --name aro-hcp-monitor --query location -o tsv) \
+                  clusterResourceId=$(az deployment group show --resource-group "${MC_RESOURCEGROUP}" --name ${MGMT_CLUSTER_NAME} --query properties.outputs.aksClusterResourceId.value -o tsv) \
+                  clusterLocation=${REGION}
```

Comment:
The result of addressing my other feedback would be that we maintain high level steps for:

* Configure `global` resource group (currently only ACR in dev, global DNS resources handled separately)
* Configure `regional` resource group
* Configure `service-cluster` resource group
* Configure `management-cluster` resource group

We may add more categories in the future, but I think this Managed Prometheus/Grafana work does fit nicely into our existing buckets